### DISCUSSION TITLE
[Enhancement] Shield Enhancement Suggestion

### DISCUSSION THREAD

**Discussion by saigyoujiyukari** [Phobos-developers/Phobos#989] [Ideas and feedback]
[SOMESHIELD]
ShieldBar.Hide=no  ; boolean. Shield bar display can now be turned off as needed, hiding both the shield bar box and shield pips.
ShieldAbilities=STRONGER,SELF_HEAL,FIREPOWER,FASTER. Lists the special attributes that this object obtains when it has this shield.
ShieldDuration=-1; integer. The duration of the shield in frames. Use -1 for infinite duration. Defaults to 0. 
[SOMEWARHEADS]
The following two tags are used together to determine how long (in frames) the affected units will be shielded by a shield.
Shield.Duration=-1; integer 
Shield.Cap=0 ; integer

**Comment by Metadorius** [comment]
1. The shield effects are basically AttachEffects from Ares, we don't plan to reimplement them.
2. The shield modification warheads are already being implemented in one of the pull requests.
